docs/concepts/v2/concepts/security-and-tokens.mdx

```diff
@@ -16,10 +16,22 @@ Both tokens can be decoded with JWT utilities, like [jwt.io](https://jwt.io).
 
 100ms client SDKs use auth tokens to [join a room](/javascript/v2/how--to-guides/set-up-video-conferencing/join).
 
-### How to get auth tokens?
 
-- No-code to get started: Get temporary auth tokens from [the dashboard](/javascript/v2/guides/token#get-a-temporary-token-from-100ms-dashboard)
-- [Set up your token server](#set-up-your-token-server): Recommended approach for production
+<FlexContainer>
+<Callout icon="shield" title="How to get auth tokens?">
+
+1. No-Code way to get started - Get temporary auth tokens from [the dashboard](/javascript/v2/guides/token#get-a-temporary-token-from-100ms-dashboard)

```

Comment:
let's plan to move this doc to the concepts page - it doesn't need to be inside a platform and aligns with your thinking on having "docs for dashboard"

Reply:
taking this up in a separate PR